dev-util/pkgconf/pkgconf-9999.ebuild

```diff
@@ -49,8 +53,10 @@ multilib_src_configure() {
 
 multilib_src_install() {
 	default
-	use pkg-config \
-		&& dosym pkgconf /usr/bin/pkg-config \
-		|| rm "${ED}"/usr/share/aclocal/pkg.m4 \
-		|| die
+
+	if use pkg-config; then
+		dosym pkgconf /usr/bin/pkg-config
+	else
+		rm "${ED}"/usr/share/aclocal/pkg.m4 || die
```

Comment:
You still have the double-slash here. My style is to prefer `"${ED%/}"/bla/foo`, as that will always be guaranteed to work and be single-slashed, even if Gentoo decides to get rid of slash-terminated `ED` some day.